app/src/main/res/layout/activity_nfc.xml

```diff
@@ -1,5 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:orientation="vertical" android:layout_width="match_parent"
-    android:layout_height="match_parent">
-</LinearLayout>
\ No newline at end of file
+    android:layout_height="match_parent"
+    android:weightSum="1"
+    tools:ignore = "UseCompoundDrawables">
+
+    <ImageView
+        tools:ignore = "UseCompoundDrawables"
+        android:id="@+id/nfc_gif"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:contentDescription="@string/descr_gif"
+        android:src="@android:drawable/sym_def_app_icon" />
+
+    <TextView
+        tools:ignore = "UseCompoundDrawables"
```

Comment:
Can you remove the spaces around the `=`? Also, do you need this ignore on every tag in this file?